cron/worker/main.go

```diff
@@ -39,13 +40,17 @@ import (
 	"github.com/ossf/scorecard/v2/cron/data"
 	"github.com/ossf/scorecard/v2/cron/monitoring"
 	"github.com/ossf/scorecard/v2/cron/pubsub"
+	sce "github.com/ossf/scorecard/v2/errors"
 	"github.com/ossf/scorecard/v2/pkg"
 	"github.com/ossf/scorecard/v2/repos"
 	"github.com/ossf/scorecard/v2/roundtripper"
 	"github.com/ossf/scorecard/v2/stats"
 )
 
-var errIgnore *clients.ErrRepoUnavailable
+var (
+	ignoreRuntimeErrors = flag.Bool("ignoreRuntimeErrors", true, "if set to true any runtime errors will be ignored")
```

Comment:
Shouldn't the default be false, and we explicitly set it to true in cron job (maybe with a FIXME) ?

Reply:
Done.